package.json

```diff
@@ -1,12 +1,14 @@
 {
-	
   "scripts": {
-    "test": "karma start karma.conf.js"
+    "test": "karma start karma.conf.js",
```

Comment:
Protractor test execution is needed to be appended to the test script either directly or via another script

Reply:
And how about integrating test coverage. It's an easy task doing it using ' nyc'

https://istanbul.js.org/docs/tutorials/